.github/workflows/daily.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js daily job
+
+on:
```

Comment:
Maybe consider adding `workflow_dispatch` so the job can also be manually run?

Reply:
done :)